app/_src/gateway/production/monitoring/health-check.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Health Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the new Node Readiness Endpoint, which provides a more reliable and simple way to determine if Kong is ready to serve user requests.
```

Comment:
1. The endpoint will only be "new" for so long, so we have concrete plans to update this doc in ~2-3 months (whenever then next release is), I suggest dropping `new` here.
2. `more reliable` sounds like a comparison, but the doc isn't actually comparing the Node Readiness Endpoint to anything.

```suggestion
This tutorial will guide you through the process of using the Node Readiness Endpoint, which provides a reliable and simple way to determine if Kong is ready to serve user requests.
```